doc/source/howtos/upgrade_guide.rst

```diff
@@ -136,71 +167,34 @@ To upgrade the cluster, follow these steps for each node:
         $ sudo apt-get remove percona-xtrabackup* percona-xtradb-cluster*
         $ sudo apt-get install percona-xtradb-cluster
 
-#. In case of Debian or Ubuntu, the ``mysql`` service starts automatically after
-   the installation. Stop the service:
-
-   .. code-block:: bash
-
-      $ sudo service mysql stop
-
 #. Back up :file:`grastate.dat`, so that you can restore it
    if it is corrupted or zeroed out due to network issue.
 
-#. Start the node outside the cluster (in standalone mode)
-   by setting the :variable:`wsrep_provider` variable to ``none``.
-   For example:
-
-   .. code-block:: bash
-
-      $ sudo mysqld --skip-grant-tables --user=mysql --wsrep-provider='none'
-
-   .. note::
-
-      To prevent any users from accessing this node while performing
-      work on it, you may add `--skip-networking
-      <https://dev.mysql.com/doc/refman/8.0/en/server-options.html#option_mysqld_skip-networking>`_
-      to the startup options and use a local socket to connect, or
-      alternatively you may want to divert any incoming traffic from
-      your application to other operational nodes.
-
-#. When the upgrade is done, stop the ``mysqld`` process.
-   You can either run ``sudo kill`` on the ``mysqld`` process ID,
-   or ``sudo mysqladmin shutdown`` with the MySQL root user credentials.
-
-   .. note:: On CentOS, the :file:`my.cnf` configuration file
-      is renamed to :file:`my.cnf.rpmsave`.
-      Make sure to rename it back
-      before joining the upgraded node back to the cluster.
 
 #. Now you can join the upgraded node back to the cluster.
```

Comment:
We should replace the text in Step5 `Now you can join the upgraded node back to the cluster`. We just upgraded packages, not the data directory. I think we should change the text something like `Now start the cluster node with 8.0 package, PXC will upgrade the data directory as needed, either as part of the startup process or a state transfer (IST/SST).`